### ISSUE TITLE
Test failing for hidemyass

### ISSUE THREAD

**Issue by stylesuxx** [chill117/proxy-lists#37]
Hey it is me again, I hope I do not annoy you too much,...

Test for hidemyass is failing:

```
  1) source.getProxies([options, ]cb) source.hidemyass should return valid proxies:
     TypeError: Cannot read property 'toString' of undefined
    at Object.<anonymous> (sources/hidemyass.js:135:48)
    at [object Object].exports.each (node_modules/cheerio/lib/api/traversing.js:300:24)
    at module.exports.parseResponseData (sources/hidemyass.js:128:12)
    at node_modules/async/dist/async.js:2718:16
    at node_modules/async/dist/async.js:2659:9
    at replenish (node_modules/async/dist/async.js:1030:17)
    at iterateeCallback (node_modules/async/dist/async.js:1015:17)
    at node_modules/async/dist/async.js:988:16
    at node_modules/async/dist/async.js:2661:13
    at node_modules/async/dist/async.js:2719:17
    at apply (node_modules/async/dist/async.js:41:25)
    at node_modules/async/dist/async.js:76:12
    at Request._callback (sources/hidemyass.js:115:4)
    at Request.self.callback (node_modules/request/request.js:187:22)
    at Request.<anonymous> (node_modules/request/request.js:1048:10)
    at IncomingMessage.<anonymous> (node_modules/request/request.js:969:12)
    at endReadableNT (_stream_readable.js:906:12)
```

I guess this is because http://proxylist.hidemyass.com/ is not showing any proxies. In this case I guess it is good that the test is  failing, since it tells us that something with the source is wrong, but still I think it should fail controlled at https://github.com/chill117/proxy-lists/blob/master/test/unit/sources/index.js#L98

**Comment by chill117** [comment]
Yes, it looks like we could catch the problem inside the hidemyass source itself, so that it doesn't throw an error like it is now.

**Comment by chill117** [comment]
Fixed by #38
